packages/babel-generator/package.json

```diff
@@ -15,6 +15,7 @@
     "babel-runtime": "^6.9.0",
     "babel-types": "^6.14.0",
     "detect-indent": "^3.0.1",
+    "jsesc": "^1.3.0",
```

Comment:
@danez is right. I’ve used jsesc@1.3.0 because it supports older versions of Node.js.